section-engineering-education/engineering-education
# Insights - Total Number of Articles Written

Added the Total number of article written insight to the author page. 

**How to test:** 
Navigate to `beta.section.io/engineering-education/` (or pull down the branch), select an author and verify that the number of articles are being displayed along with an icon just below the author bio. The number should also correlate to the number of articles listed below - screenshot below. 

Hovering over the icon and text should also display a tooltip describing the item. 

beta.section.io credentials can be found in last pass.

<img width="1072" alt="image" src="https://user-images.githubusercontent.com/15935329/161125533-852b8c0e-9d95-4939-b550-9027998db2d6.png">
